pgcli/main.py

```diff
@@ -88,6 +91,8 @@ def __init__(self, force_passwd_prompt=False, never_passwd_prompt=False,
         smart_completion = c['main'].as_bool('smart_completion')
         completer = PGCompleter(smart_completion, pgspecial=self.pgspecial)
         self.completer = completer
+        self._completer_lock = threading.Lock()
+        self._restart_completion = threading.Event()
```

Comment:
This doesn't seem to be used. (`completion_refresher` initializes its own Event)